README.md

```diff
@@ -1,34 +1,34 @@
+7. Create a PR from your `feature/personal branch` into `staging` branch verifying that the publication date in the post's file name is still correct, provide a link to your post on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/) and add @lauracowen, SMEs, and/or other reviewers to get their final approval for both content and format.  To make any changes, update the PR with new commit and repeat step 5 & 6 until all issues are resolved.  Once approved, @lauracowen (or another admin) will merge the PR into `staging` and shepherd the post through the remaining steps.
 8. Request a build of the [staging site](https://staging-openlibertyio.mybluemix.net/blog/) (same as step 5)
     1. Go to [Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io)
     2. More Options > Trigger Build > Make sure the `master` branch is selected (default) > Trigger custom build
 9. Once the build completes, check to make sure the blog renders correctly on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/).  This is the final check before the post goes into the [production site](https://openliberty.io/blog/).
 10. If there are issues found on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/), they must be resolved quickly, either by merging a new PR with a fix, or reverting it.  If you pushed a new PR with the fix, make sure it also gets pushed to the `draft` branch.
-11. Create a PR from `staging` branch into `prod` branch an admin approver.  Once approved, merge into `prod`.
-12. Rebuild the [production site](https://openliberty.io/blog/) on IBM Cloud and verify the post looks as expected on openliberty.io
+11. Create a PR from `staging` branch into `prod` branch (with an admin approver, though this check is currently purposefully disabled).  Once approved, merge into `prod`.
+12. Rebuild the [production site](https://openliberty.io/blog/) on IBM Cloud and verify the post looks as expected on openliberty.io.  If the post used a future date, it will not render on the production site until that day or later (and will need the site to be rebuild to publish).
+13. Now that the contents of the feature branch has been successfully delivered to & tested on `draft`, `staging`, and `prod`, delete the feature branch.
 
 Summary of commit flow through branches:  
-`prod` <-PR- `staging` <-PR- `draft` <-PR- `personal branch from fork`  
-In other words, `prod` should always be a commit subset of `staging` which should be a commit subset of `draft`.  Note, however, that you're not actually, normally, doing a PR from `draft` to `staging` but instead from your `personal branch from fork` into `staging`, but only after it already has been in `draft`.
+`prod` <-PR- `staging` <-PR- `draft` <-PR- `feature/personal branch from fork`  
```

Comment:
Can we make it more clear that we shouldn't make pr's from draft into staging into prod? Maybe something like:
feature/personal branch -> draft
test on draft site
feature/personal branch -> staging
final testing and Laura's/SME's review
feature/personal branch -> prod
Build Ol.io

Reply:
Yes, this section does need a better flow diagram and I might just remove it in the interim, but as a side note, I would expect a PR from staging into prod, and not from the feature/personal branch into prod.  I think prod should only ever have PRs from staging, with the exception being some extremely time-sensitive fix of something that's already live on the prod site.

Reply:
That's true about staging->prod